src/app/gateways/views/BuiltinGatewayDataplanesView.vue

```diff
@@ -17,8 +17,27 @@
         :src="`/meshes/${route.params.mesh}/mesh-gateways/${route.params.gateway}`"
         v-slot="{ data: meshGateway, error }: MeshGatewaySource"
       >
-        <div class="stack">
+        <div
+          class="stack"
+        >
           <KCard>
+            <search>
+              <FilterBar
```

Comment:
no diff between this and https://github.com/kumahq/kuma-gui/pull/3089/files?w=1#diff-fa7b9859eb3424e3606790db4f7fb3c6fa91d418504f3219b0d27b07ab084f64L54-L71